routes/selection.js

```diff
@@ -58,6 +58,11 @@ router.get('/:geotype/:geoid', async (req, res) => {
         status:  [`Failed to find selection for hash`],
       });
     }
+  } else if (!Object.keys(summaryLevels).includes(geotype)) {
+    console.error(`${geotype} is not a valid geotype.`);
+    res.status(500).send({
```

Comment:
Might be a good idea o change this to a 400 Bad Request response.

Reply:
@horatiorosa , changes made